documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,99 @@
+
+### Knotx publish start command
+`java -Dlogback.configurationFile=/content/knotx/publish/logback.xml -Dvertx.cacheDirBase=/content/knotx/publish/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=18093 -Dcom.sun.management.jmxremote.rmi.port=18093 -Djava.rmi.server.hostname=10.254.165.236 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/publish/knotx.jar -conf /content/knotx/publish/config/publish-config.json`
+
+### Knotx mocks start command
+`java -Dlogback.configurationFile=/content/knotx/mocks/logback.xml -Dvertx.cacheDirBase=/content/knotx/mocks/.vertx -Xms1024m -Xmx2048m -XX:ReservedCodeCacheSize=128m -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=19000 -Dcom.sun.management.jmxremote.rmi.port=19000 -Djava.rmi.server.hostname=10.254.165.163 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Xloggc:/content/logs/knotx/gc.log -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=5M -jar /content/knotx/mocks/knotx.jar -conf /content/knotx/mocks/config/mocks-config.json`
+
+### Test harness
+* 1 x Jmeter Master - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+* 2 x Jmeter Servers - c4.xlarge, 4CPU, 8GB RAM, 750Mbps bandwidth,centos 6.8.
+
+### Repositories
+* mocks-knotx - \knotx-performance-tests\config-knotx\
+* config-knotx - \knotx-performance-tests\mocks-knotx\
+
+## Tools
+* JMeter 3.1 with plugins (at AWS) as a Load Generator.
+* Zabbix 3.2.1 (Internal instance at Cognifide Office) as an Server Monitoring.
+* Influx (Internal Instance at AWS set up on JMeter Master machine) as a Live Test Monitoring database.
+* Grafana 3.0.1 (Internal Instance at Cognifide Office) as a Live Test Monitoring tool.
+
+## Test Process
+Test process that we used is based on Microsoft Performance testing process - https://msdn.microsoft.com/en-us/library/bb924376.aspx.
+
+## Test Scripts
+* Test script implemented in JMeter 3.1.
+* Think time 4-6 seconds (Gaussian Random Timer).
```

Comment:
Do you think we should add some definitions or urls to definitions here?
E.g. [Think time](http://www.webperformance.com/library/dictionary.html#thinktime) ?